build-tests/node-library-build-tslint-test/.npmignore

```diff
@@ -0,0 +1,24 @@
+# Ignore everything by default
+**
```

Comment:
Probably should just ignore everything and mark this project as private.

Reply:
This is a good idea, but it applies to many projects.  We should handle it in a separate PR.